src/commands/unity-symbols/helpers.ts

```diff
@@ -0,0 +1,22 @@
+import {getBaseSourcemapIntakeUrl} from '../../helpers/base-intake-url'
+import {RequestBuilder} from '../../helpers/interfaces'
+import {MultipartPayload, upload, UploadOptions} from '../../helpers/upload'
+import {getRequestBuilder} from '../../helpers/utils'
+
+export const getUnityRequestBuilder = (apiKey: string, cliVersion: string, site: string) =>
+  getRequestBuilder({
+    apiKey,
+    baseUrl: getBaseSourcemapIntakeUrl(site),
+    headers: new Map([
+      ['DD-EVP-ORIGIN', 'datadog-ci unity-symbols'],
```

Comment:
If I remember correctly, intake will drop header if it contains spaces, you may want to check if it is still the case.

Reply:
🤔 gee I hope not as Flutter does this too...

Reply:
everything in `datadog-ci` is doing this actually.